x/btcstaking/types/multi_staking_allow_list_test.go

```diff
@@ -0,0 +1,73 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/stretchr/testify/require"
+
+	"github.com/babylonlabs-io/babylon/v3/x/btcstaking/types"
+)
+
+func TestLoadMultiStakingAllowList(t *testing.T) {
```

Comment:
Ideally we would have some test checking that neither of allowlist (mainnet, testnet, default) contains duplicates as duplicate can mean that somebody did make mistake while creating the allowlist 🤔